fearless/Modules/Staking/StakingUnbondSetup/StakingUnbondSetupViewController.swift

```diff
@@ -0,0 +1,207 @@
+import UIKit
+import SoraFoundation
+import CommonWallet
+
+final class StakingUnbondSetupViewController: UIViewController, ViewHolder {
+    typealias RootViewType = StakingUnbondSetupLayout
+
+    let presenter: StakingUnbondSetupPresenterProtocol
+
+    var selectedLocale: Locale {
+        localizationManager?.selectedLocale ?? .autoupdatingCurrent
+    }
+
+    init(
+        presenter: StakingUnbondSetupPresenterProtocol,
+        localizationManager: LocalizationManagerProtocol
+    ) {
+        self.presenter = presenter
+
+        super.init(nibName: nil, bundle: nil)
+
+        self.localizationManager = localizationManager
+    }
+
+    var uiFactory: UIFactoryProtocol = UIFactory()
```

Comment:
private let?

Reply:
The purpose here is to override the property if we will need in future